core/src/components/banner/tds-banner.scss

```diff
@@ -74,14 +74,14 @@
   }
 
   .banner-close {
-    color: var(--sdds-banner-x-color);
+    color: var(--tds-banner-x-color);
 
     button {
       padding-right: 16px;
       padding-top: 14px;
       background-color: transparent;
       border: none;
-      color: var(--sdds-banner-x-color);
+      color: var(--tds-banner-x-color);
 
       &:hover {
         cursor: pointer;
```

Comment:
I could not get there, but please check line 94 - Maybe mixing could be used or create a local variable for focus for this component. The banner should not rely on focus variables from other components.

Reply:
should be fixed now